db/model/lens.js

```diff
@@ -137,6 +127,29 @@ module.exports = function lens(seq, dataTypes) {
         }
       },
 
+      afterDestroy(inst /* , opts */) {
+        redisCache.del(inst.id);
+        redisCache.del(inst.name);
+      },
+
+      afterCreate(inst /* , opts */) {
+        // require inside hook to remove cyclic dependency.
+        const cleanAndCreateLensJson = require(
+        '../../api/v1/controllers/lenses'
+        ).cleanAndCreateLensJson;
+
+        const lensObj = cleanAndCreateLensJson(inst);
```

Comment:
I'd like to avoid having any dependencies *from* db layer *into* api layer--it can get very messy!